src/actions/postActions.js

```diff
@@ -1,4 +1,10 @@
 // * posts have an title and body
 // * you can create a post
 // * you can delete a post
-// * BONUS: you can update a post by passing an id and new body
\ No newline at end of file
+// * BONUS: you can update a post by passing an id and new body
+export const ADD_POST = 'ADD_POST';
+
+export const addPost = post => ({
```

Comment:
Each post will have a title and a body, so you'll want to incorporate that here. Maybe deconstructing post to include them, and then changing your payload accordingly. :)